### ISSUE TITLE
No results using trained model.

### ISSUE THREAD

**Issue by saikia1** [nicolasmetallo/car-damage-detector#2]
I have trained the model for 10 epochs and have used the trained model to run the command to plot splash. But it doesn't output anything, detected mask only have zeros inside it.

**Comment by jos1337** [comment]
same here, any solution or ideas?

**Comment by surajitdeb1** [comment]
Getting the same issue. Anyone able to resolve this?

**Comment by dexception** [comment]
Change the requirement.txt and instead of tensorflow use tensorflow-gpu

**Comment by suneetbhatia** [comment]
Hi, I have trained the model for 5 epocs on the given training data . But cant find any output/mask on either test or train images. I was expecting atleast some result even if it was not as accurate. But I am just getting the input image back.

**Comment by achieverprince** [comment]
same here!!!

**Comment by SinaArdehali** [comment]
Does anyone has found a solution for that ? i am also getting the input image back
